@navikt/core/react/src/index.ts

```diff
@@ -1,3 +1,4 @@
+"use client";
```

Comment:
Er enklere å bare sette "use-client" her vs å sette det i alle filer. Ser for meg det raskt kan skje feil da hvis man glemmer å sette det en plass. Har ikke funnet en ESLint-plugin som sjekker om en fil bør ha "use-client" enda, men har kanskje funket da